src/csharp/Pester/Pester.csproj

```diff
@@ -1,14 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.1;net48</TargetFrameworks>
```

Comment:
Using net452 is deliberate to be compatible with powershell 3.

Reply:
Thanks for clarification.  Will revert